Task for creating GitHub deployments (#258)
tektoncd/catalog: task/github-create-deployment/0.1/README.md

# GitHub

A collection of tasks to help working with the [GitHub

> API](https://developer.github.com/v3/).

Reviewer: I guess we can remove this now as every task has it's own README

## GitHub token

> Most tasks would expect to have a secret set in the kubernetes secret `github`

Reviewer 2: ```suggestion
This task expect to have a secret set in the kubernetes secret `github`
```

Reviewer 2: (probably need to fix the wording of the other github tasks* who got moved to their own directory)

with a GitHub token in the key `token`, you can easily create it on the
command line with `kubectl` like this :

```
kubectl create secret generic github --from-literal token="MY_TOKEN"
```

## Create a GitHub deployment

The `github-create-deployment` task lets you create a GitHub deployment.

> See GitHub's deployment API on [creating a deployment](https://developer.github.com/v3/repos/deployments/#create-a-deployment) for more information.

Reviewer: Probably we can move this to the starting of README file

### Install the Task

```

> kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/github/create_deployment.yaml

Reviewer: ```suggestion
kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/task/github-create-deployment/0.1/github-create-deployment.yaml
```

Author: Thanks! Totally missed that.
